### DISCUSSION TITLE
Records Missing while using MysqlHook

### DISCUSSION THREAD

**Discussion by Killer2499** [apache/airflow#30995] [General]
Hello,

I am trying using MysqlHook to fetch results from a query but sometimes I see there is a difference between what actually exists and what is returned. One record gets missing in the result set . Did any one face similar issue or do we have any open issue?

**Comment by potiuk** [comment]
You shoudl check yourself - it should be easy by searching, there is nothing else anyone else can do about it- just searching. From the top of my head, I don't recall any but you should check it. And I seriously doubt it is something in the implementation - MySQL hook uses common sql hook. As usual attempting to upgrade to latest version is the first advice (you can upgrade providers independently from Airlfow) and then if you see this error and can somewhat reproduce it - reporting an issue with all the details, logs, versions etc. is the way to go.
